src/frontend/src/test-e2e/pinAuth.test.ts

```diff
@@ -61,3 +110,63 @@ test("Should not prompt for PIN after deleting temp key", async () => {
     await FLOWS.login(userNumber, DEVICE_NAME1, browser);
   }, APPLE_USER_AGENT);
 }, 300_000);
+
+test("Log into client application, after PIN registration", async () => {
```

Comment:
I find this description confusing. Maybe `"Log into client application using PIN registration flow"`?